common/next/next.config.js

```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-const withTM = require('@weco/next-plugin-transpile-modules');
+const withTM = require('next-transpile-modules')(['@weco']);
```

Comment:
is this what fixes the hot reloading?

Reply:
Could have been, I thought it was the removal of `getInitialProps` for the `app`. This was done as we don't maintain our version any more.